rubyforgood/human-essentials
# Change State to Status on Distribution Table

Resolves #4603 

### Description

- Changes the state column to status
- Enums have been changed
- Test example descriptions have been changed
- Changes on export made

### Type of change



* New feature (non-breaking change which adds functionality)

### How Has This Been Tested?
- Has been tested locally with all unit tests passing
- Has been verified on the FE that it shows Status
- Has been verified by downloading distributions csv that it shows as status on csv as well

#### State Renaming on FE test steps:
- Login as Bank User with email: `user_1@example.com`
- Navigate to Distributions
- Check Column is being shown as status

#### State Renaming on CSV download test steps:
- Login as Bank User with email: `user_1@example.com`
- Navigate to Distributions
- Export Distributions
- Verify that the CSV shows the column Status



### Screenshots

- Screenshots for verification:
<img width="1154" alt="image" src="https://github.com/user-attachments/assets/68d8db54-fe33-4bd2-bb6e-20c6d7d8baa0">

<img width="1040" alt="image" src="https://github.com/user-attachments/assets/13a03433-2857-424c-a785-0c69fe50fa0b">
